src/Compiler/Service/FSharpCheckerResults.fs

```diff
@@ -2089,6 +2089,36 @@ type internal TypeCheckInfo
                 Trace.TraceInformation(sprintf "FCS: recovering from error in GetSymbolUseAtLocation: '%s'" msg)
                 None)
 
+    member _.GetSymbolUsesAtLocation(line, lineStr, colAtEndOfNames, names) =
```

Comment:
Can you also de-duplicate thsi with the member right above?
If I did not overlook it, it is again the same thing (just differing in either list or option as return value)